homeassistant/components/iss/binary_sensor.py

```diff
@@ -56,42 +63,31 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class IssBinarySensor(BinarySensorEntity):
     """Implementation of the ISS binary sensor."""
 
+    _attr_device_class = DEFAULT_DEVICE_CLASS
+
     def __init__(self, iss_data, name, show):
         """Initialize the sensor."""
         self.iss_data = iss_data
         self._state = None
-        self._name = name
+        self._attr_name = name
         self._show_on_map = show
-
-    @property
-    def name(self):
-        """Return the name of the sensor."""
-        return self._name
-
-    @property
-    def is_on(self):
-        """Return true if the binary sensor is on."""
-        return self.iss_data.is_above if self.iss_data else False
-
-    @property
-    def device_class(self):
-        """Return the class of this sensor."""
-        return DEFAULT_DEVICE_CLASS
+        self._attr_is_on = self.iss_data.is_above if self.iss_data else False
 
     @property
     def extra_state_attributes(self):
         """Return the state attributes."""
         if self.iss_data:
             attrs = {
-                ATTR_ISS_NUMBER_PEOPLE_SPACE: self.iss_data.number_of_people_in_space,
-                ATTR_ISS_NEXT_RISE: self.iss_data.next_rise,
+                "number_of_people_in_space": self.iss_data.number_of_people_in_space,
+                "next_rise": self.iss_data.next_rise,
```

Comment:
Why did you remove these constants?

Reply:
For what reason we need to look far above for a constant that is only used once?

Reply:
so? 🤷‍♂️

Reply:
I suppose it's less readable that way, why make a constant that is only used once, and is literally spells the same?

Reply:
We generally use constants for keys (as much as possible), this feels it is going in the opposite direction.

Anyways, the first other review comment is way more important than this one, as that actually breaks the integration.